syntax/pico8.vim

```diff
@@ -49,7 +49,7 @@ syn match  luaError "\<\%(end\|else\|elseif\|then\|until\|in\)\>"
 syn region luaFunctionBlock transparent matchgroup=luaFunction start="\<function\>" end="\<end\>" contains=ALLBUT,luaTodo,luaSpecial,luaCond,luaCondElseif,luaCondEnd,luaRepeat
 
 " if then else elseif end
-syn keyword luaCond contained else
+syn keyword luaCond contained else and or
```

Comment:
Sorry, missed this.  This includes highlighting for the `and` and `or` keywords (which are valid in lua as well as pico8's lua flavor.